scan/tasks.py

```diff
@@ -49,6 +53,35 @@ def task_cmd():
         if snr_master :
             logger.info("SNR Master Data Received")
 
+    """
+    ######### Update Bootstrap Peers ###########
+        Best to run this task instead of having the page load it each time it's opened. 
+        Also exports to env for peer scan use. 
+    """
+    if AUTO_BOOTSTRAP_PEERS:
+        bootstrap_peers = BRS_BOOTSTRAP_PEERS
+        caching_peers = CachingBootstrapNodes()
+        bootstrap_network = os.environ.get("BRS_BOOTSTRAP_NETWORK", ".signum.network")
+        if bootstrap_network is "": bootstrap_network = ".signum.network"
+        try:
```

Comment:
> Try...except is not needed here, because if query `auto_bootstrap_peers` finds nothing, then `list(auto_bootstrap_peers) = []`

Although not explicitly required, IF someone adds BRS_BOOTSTRAP_NETWORK= (without anything) in their env file then it will fail hitting the except. This is simply for easy adoption for less experienced operators.